Source/Admin/Web/ProjectController.cs

```diff
@@ -51,7 +51,7 @@ public void Post([FromBody] CreateProject command)
                 Id = command.Id,
                 NationalSocietyId = command.NationalSocietyId,
                 DataOwnerId = command.DataOwnerId,
-                SurveillanceContex = command.SurveillanceContex
+                //SurveillanceContex = command.SurveillanceContex
```

Comment:
Could you please add a comment on why SurveillanceContext is commented out? Should it be removed?